.github/workflows/benchmark_large.yml

```diff
@@ -9,6 +9,7 @@
 name: Benchmark Large
 
 on:
+  pull_request:
```

Comment:
Revert before merge. You can aslo use workflow_dispatch for testing, or have this run on pull requests that only satisfy certain conditions (e.g. paths modified, branch names matching a regex), https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request.